docs/operator-guide/cd/deploy-application-in-remote-cluster-via-irsa.md

```diff
@@ -0,0 +1,682 @@
+---
+
+title: "Deploy Application In Remote Cluster via IRSA"
+sidebar_label: "Deploy Application In Remote Cluster via IRSA"
+description: "Learn how to securely deploy applications across AWS accounts using IRSA (IAM Roles for Service Accounts) in KubeRocketCI. Step-by-step guide for EKS cross-account access configuration."
+
+---
+<!-- markdownlint-disable MD025 -->
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploy Application In Remote Cluster via IRSA
+
+<head>
+  <link rel="canonical" href="https://docs.kuberocketci.io/docs/operator-guide/cd/deploy-application-in-remote-cluster-via-irsa" />
+</head>
+
+KubeRocketCI provides the capability to deploy applications securely using IAM Roles for Service Accounts (IRSA) in AWS EKS.
+This integration enables Kubernetes pods to assume IAM roles for secure and temporary access to AWS resources, eliminating the need for long-lived credentials. While the deployment process is streamlined for most users, the platform also supports advanced configurations for custom permissions and role management, ensuring flexibility for more complex scenarios.
+
+## Prerequisites
+
+To start using this approach, you need to have OIDC (OpenID Connect) already configured for your EKS cluster. This setup allows Kubernetes service accounts to securely assume IAM roles. For your convenience, please follow our documentation [EKS OIDC With Keycloak](../operator-guide/auth/configure-keycloak-oidc-eks.md). This setup seamlessly integrates OIDC with minimal effort.
+
+## Roles
+
+Cross-account interaction is performed through IRSA with a two-tiered IAM role setup:
+
+- In AWS Account A, the EKS cluster runs a kuberocketci cd-pipeline-operator with service account.
+- This service account obtains temporary credentials through IRSA, which are associated with the CD-Pipeline-Operator-IRSA role.
```

Comment:
Do we have this terraform code already? Maybe let's call role: AWSIRSA_{cluster_name}_CDPipelineOperator so to leverage our common pattern, cc @MykolaMarusenko